tools/toolchain/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/fedora:34
+FROM docker.io/fedora:37
```

Comment:
This container is several gigabytes in size, since it contains java-1.8.0-openjdk which pulls a bunch of X stuff.

Reply:
It's not about size, but reduce the no. of packages that may break, consume resources, security (for a secure build process, not product). The cost will be in installing some deps explicitly - not sure it's a bad thing (good for SBOM)

Reply:
I doubt it will reduce anything.

```console
$ ./tools/toolchain/dbuild rpm -qa | wc -l
1001
$ podman run --rm docker.io/fedora:37 rpm -qa | wc -l
149
```

Anything you remove will be brought back by the huge dependency list we have.